airflow/exceptions.py

```diff
@@ -338,3 +338,17 @@ class TaskDeferralError(AirflowException):
 
 class PodReconciliationError(AirflowException):
     """Raised when an error is encountered while trying to merge pod configs."""
+
+
+class RemoveInAirflow3DeprecationWarning(DeprecationWarning):
```

Comment:
```suggestion
class RemovedInAirflow3DeprecationWarning(DeprecationWarning):
```

Nitpick: Make this a past-tense description of the warning, instead of a command. This makes it consistent with its docstring and other exceptions in `airflow.exception`.

Reply:
https://github.com/apache/airflow/pull/25799/commits/583d6c1b888c4a3fb33543e80f842a4103c5aa5c